src/main/java/seedu/address/MainApp.java

```diff
@@ -50,6 +51,7 @@
     protected ItemModel model;
     protected Config config;
     protected ElisaStateHistory stateHistory;
+    protected JokeList jokeList;
```

Comment:
I just feel like this should not be done at the MainApp. Instead maybe move the jokeList to the model and do everything from the model?

Reply:
done!